### DISCUSSION TITLE
Reg : who all are using my public repository in Github

### DISCUSSION THREAD

**Discussion by jagadeeshdevula** [community/community#22783] [New to GitHub]
<p>Hi,<br>
I want to see on my Linux machine who all are using my public repository in Github.<br>
Except for the Github-Dorks tool.<br>
Is it possible?<br>
If, Yes please tell me those tool names.</p>
<p>Thanks.</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/jagadeeshdevula">@jagadeeshdevula</a>,</p>
<p>There is no GitHub method to get this information. Repositories can be downloaded without having a GitHub account, so this data would not be possible to obtain.</p>
